### PR TITLE
translation: Update algorithms_are_everywhere.md

### PR DESCRIPTION
**Pull request by K3v123**
I made changes to yuelinxin's suggestions.
again thank you so much for your suggestion yuelinxin!

If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [ ] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

File: docs-en/chapter_introduction/algorithms_are_everywhere.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ When we hear the word "algorithm", we naturally think of mathematics. However, m
...
 
-**Example 1: Looking Up a Dictionary**. In a standard dictionary, each word corresponds to a phonetic transcription and the dictionary is organized alphabetically based on these transcriptions. Let's say we're looking for a word that begins with the letter $r$. This is typically done in the following way:
+**Example 1: Looking Up a Dictionary**. In an English dictionary, words are listed alphabetically. Suppose we're searching for a word that starts with the letter $r$. This is typically done in the following way:
 
 1. Open the dictionary around its midpoint and note the first letter on that page, assuming it to be $m$.
-2. Given the sequence of words following the initial letter $m$, estimate where words starting with the letter $r$ might be located within the alphabetical order.
+2. Since $r$ comes after $m$ in the pinyin alphabet, we exclude the first half of the dictionary and narrow the search to the second half.
```
Comment (krahets) [inline docs-en/chapter_introduction/algorithms_are_everywhere.md:10]: pinyin alphabet -> alphabet

**Review by krahets — COMMENTED** [review]
Thanks for the optimization! Please address the comment.

**Review by yuelinxin — COMMENTED** [review]
Same modification mentioned by @krahets

File: docs-en/chapter_introduction/algorithms_are_everywhere.md (hunk 1 above)
Comment (K3v123) [reply docs-en/chapter_introduction/algorithms_are_everywhere.md]: err... my bad, and thank you so much for your comment Krahets!

**Review by yuelinxin — APPROVED** [review]
Great work, thank you!

**Review by krahets — APPROVED** [review]
Thanks for the PR!

**Pull request merged**